site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -316,6 +316,103 @@ or SDK.WatchGameServer() instead to list the connected players.
 
 [playerstatus]: {{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.PlayerStatus" >}}
 
+### Counters And Lists
+
+{{< alpha title="Counters And Lists" gate="CountersAndLists" >}}
+
+The `Counters` and `Lists` features in the SDK offer a flexible configuration for tracking various entities like 
+players, rooms, and sessions. However, it's important to note that this feature has been in Alpha stage and disabled
+by default.
+
+{{< alert title="Note" color="info">}}
+There is a potential race condition when count values are set from both the SDK and through the K8s API(Allocation or otherwise),
+since the SDK update operation back to the CRD value is batched asynchronous any value incremented past the capacity will be
+silently truncated to capacity.
+
+When retrieving count values through `get` operations, it's important to note that the values may not always represent
+the most up-to-date value. Changes to Counters and Lists can occur through Game Server Allocation Actions as well as the SDK, 
+so this value may not always be accurate as it could be reading cached data. However, value will be
+eventually consistent when coming back to the SDK.
+{{< /alert >}}
+
+### Counters
+
+#### Alpha().GetCounterCount(key)
+
+This function retrieves the current count for a specified Counter in the game server. This returns `-1` an `error` if 
+the key is not predefined.
+
+#### Alpha().SetCounterCount(key, amount)
+
+This function sets the count to a given value. This operation overwrites any previous values and the new value cannot
+exceed the Counter's capacity. The function returns `true` if successful, or `false` along with an `error` if the operation fails.
+
+#### Alpha().IncrementCounter(key, amount)
+
+This function increments the count of a specified Counter by a given nonnegative value amount. The function returns `false`
+and an `error` if the key wasn't predefined in the GameServer resource and returns or if the Counter is already at capacity,
+indicating no increment will occur. If the update is successful this returns `true`.
+
+#### Alpha().DecrementCounter(key, amount)
+
+This function decreases the count of a specified Counter by a given nonnegative amount. It ensures the Counter's count does
+not go below zero and operates against the current CRD value. It returns `false` and an `error` if the Counter's count is
+already at zero, if the decrement amount is negative, or if there is another error, indicating no decrement will occur.
+
+#### Alpha().SetCounterCapacity(key, amount)
+
+This function sets the maximum capacity for a specified Counter. A capacity value of 0 indicates no capacity limit.
```

Comment:
"A capacity value of 0 indicates no capacity limit" - don't we have a default capacity limit in this case?

Reply:
If I am correct, a capacity value of 0 indicates that there is no limit to the Counter's capacity. Please correct me if I am wrong

Reply:
`[https://github.com/googleforgames/agones/issues/2716#:~:text=](https://github.com/googleforgames/agones/issues/2716#:~:text=FleetAutoscaling)A%20capacity%20of%200%20is%20no%20capacity`